docs/Contributing/API-for-contributors.md

```diff
@@ -2656,6 +2657,28 @@ Signals the Fleet server to send a webbook request with the device UUID and seri
 
 ---
 
+#### Trigger Filevault key escrow
+
+Sends a signal to the Fleet Desktop to initiate a Filevault key escrow. This is useful for setting the escrow key initially as well as in scenarios where a token rotation is required. **Requires Fleet Premium license**
```

Comment:
I think this should just be "Fleet Desktop" instead of "the Fleet Desktop"

Reply:
agreed! change made.